src/model/CoilCoolingDXSingleSpeedThermalStorage.cpp

```diff
@@ -2108,6 +1977,10 @@ namespace model {
     ok = setDischargeOnlyModeRatedCOP(3.0);  // IDD default
     OS_ASSERT(ok);
 
+    ok = setAncillaryElectricPower(0.0);
+    OS_ASSERT(ok);
+    ok = setColdWeatherOperationAncillaryPower(0.0);
+    OS_ASSERT(ok);
```

Comment:
(now that I've requested it, I hope we don't find out later that E+ treats blank and zero differently... sometimes that happens)